zds/forum/models.py

```diff
@@ -280,11 +296,55 @@ def last_read_post(self):
         except TopicRead.DoesNotExist:
             return self.first_post()
 
+    def resolve_last_read_post_absolute_url(self):
+        """resolve the url that leads to the last post the current user has read. If current user is \
+        anonymous, just lead to the thread start.
+
+        :return: the url
+        :rtype: str
+        """
+        user = get_current_user()
+        if not user.is_authenticated():
```

Comment:
`if not user` ou `user is None` revient au même, mais la norme te conseille le premier ;)